pkg/repositories/config/database.go

```diff
@@ -21,10 +24,12 @@ type DbConfigSection struct {
 // DbConfig is database config. Contains values necessary to open a database connection.
 type DbConfig struct {
 	BaseConfig
-	Host         string `json:"host"`
-	Port         int    `json:"port"`
-	DbName       string `json:"dbname"`
-	User         string `json:"user"`
-	Password     string `json:"password"`
-	ExtraOptions string `json:"options"`
+	Host         string                  `json:"host"`
```

Comment:
you should be using this right
https://github.com/flyteorg/flytestdlib/blob/master/database/config.go